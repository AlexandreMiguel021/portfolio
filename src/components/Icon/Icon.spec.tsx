import { render } from '@testing-library/react'
import { Icon } from '.'
import '@testing-library/jest-dom'

describe('Icon', () => {
	it('should render a Icon correctly', () => {
		const { container } = render(<Icon href="/content">Content</Icon>)
		expect(container).toMatchSnapshot()
	})
	it('should render an Icon with child content and href attribute', () => {
		const { getByText } = render(<Icon href="/content">Content</Icon>)
		expect(getByText('Content')).toHaveAttribute('href')
	})
})
