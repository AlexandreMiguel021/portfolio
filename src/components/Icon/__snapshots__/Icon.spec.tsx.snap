// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon should render a Icon correctly 1`] = `
<div>
  <a
    class="icon"
    href="/content"
    target="_blank"
  >
    Content
  </a>
</div>
`;
